pkg/internal/cryptosetup/cryptosetup_test.go

```diff
@@ -34,8 +34,11 @@ func TestPrepareJWECrypto(t *testing.T) {
 		keyHandleToBeCreated, err := keyset.NewHandle(mac.HMACSHA256Tag256KeyTemplate())
 		require.NoError(t, err)
 
+		testMACValue := []byte("testValue")
+		mockCrypto := crypto.Crypto{ComputeMACValue: testMACValue}
```

Comment:
you can use directly the default crypto implementation here:
`c, err := tinkcrypt.New()`